bot/resources/tags/windows-path.md

```diff
@@ -1,10 +1,10 @@
 **PATH on Windows**
 
-If you have installed Python but you forgot to check the *Add Python to PATH* option during the installation you may still be able to access your installation with ease.
+If you have installed Python, but you forgot to check the *Add Python to PATH* option during the installation, you may still be able to access your installation with ease.
 
-If you did not uncheck the option to install the Python launcher then you will find a `py` command on your system. If you want to be able to open your Python installation by running `python` then your best option is to re-install Python.
+If you did not uncheck the option to install the Python launcher then you will find a `py` command on your system. If you want to be able to open your Python installation by running `python`, then your best option is to re-install Python.
 
-Otherwise, you can access your install using the `py` command in Command Prompt. Where you may type something with the `python` command like:
+Otherwise, you can access your installation using the `py` command in Command Prompt, where you may type something with the `python` command such as:
```

Comment:
```suggestion
Otherwise, you can access your installation using the `py` command in Command Prompt. Where you may type something with the `python` command such as:
```
The change regarding the comma did not make any improvement.

Reply:
Having `Where` as the next sentence doesn't make any sense to me here. Perhaps a semi-colon would be more appropriate?

Reply:
Yeah this one didn't feel "right" fully. "Where" itself may not be the right word?

I don't think I fully understand this sentence either. Is this sentence meant to convey that `py` and `python` are interchangable?

Reply:
I believe it's saying you can use `py` wherever you'd use `python` in the command line.

Perhaps `Otherwise, you can access your installation using the `py\` command in Command Prompt rather than `python`.

Reply:
In fact, after re-reading the tag I realise this sentence is rather redundant as the previous sentence covers it, so I'll ignore for now